src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -527,8 +531,12 @@ private BoundExpression GetExtensionMemberAccess(SyntaxNode syntax, BoundExpress
                 case NamedTypeSymbol namedTypeSymbol:
                     bool wasError = false;
 
-                    return BindTypeMemberOfType(receiver, namedTypeSymbol.Name, namedTypeSymbol,
-                        syntax, right: syntax, diagnostics, ref wasError);
+                    if (!typeArgumentsOpt.IsDefault)
+                    {
+                        namedTypeSymbol = ConstructNamedTypeUnlessTypeArgumentOmitted(syntax, namedTypeSymbol, typeArgumentsSyntax, typeArgumentsOpt, diagnostics);
```

Comment:
>ConstructNamedTypeUnlessTypeArgumentOmitted [](http://example.com/codeflow?start=42&length=43)

Are we testing scenario with omitted type arguments (```<,>```)? We probably should create an unbound generic type in that case.

Reply:
Yes (see tests with `_OmittedTypeArgument`). Indeed, we produce a unbound generic symbol then.